content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -11,11 +11,22 @@ This page shows how to configure a Key Management Service (KMS) provider and plu
 
 * {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
+{{< tabs name="tab_with_md" >}}
+{{% tab name="v1" %}}
 * Kubernetes version 1.10.0 or later is required
 
 * etcd v3 or later is required
 
```

Comment:
Should a KMS provider be required along with a KMS provider plugin? 
This is stating the obvious but at least it will be clear.